### PR TITLE
Note in free gas networks about EIP-1820 requiring a small gas amount

### LINKED ISSUE #280: Update 0 gas networks topic to cover predefined contracts requiring gas
https://besu.hyperledger.org/en/stable/HowTo/Configure/FreeGas/ needs updating to cover that some predefined contracts might require gas. 

This is due to the EIP:
https://eips.ethereum.org/EIPS/eip-1820
The raw transaction includes payment of gas to make this work, and the docs on the EIP point out that there is a need to transfer a limited amount of gas to the owner of the account to make this work.

### PR DESCRIPTION
**Pull request by grantnoble**
Plus a typo in a preceding para.
Fixes #280 
Signed-off-by: grantnoble <grant.noble@consensys.net>

### CODE REVIEW

File: docs/HowTo/Configure/FreeGas.md [hunk 1]
```diff
@@ -12,15 +12,21 @@ The miner (or validator in PoA networks) that includes the transaction in a bloc
...
+to limit resource use.
 
 !!! tip
 
     We use the term _free gas network_ to refer to a network with a gas price of zero. A network
     with a gas price of zero is also known as a _zero gas network_ or _no gas network_.
 
+!!! note
+
+    Networks implementing the smart contract outlined in
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/FreeGas.md:26]: Suggest removing outlined as it's not needed

File: docs/HowTo/Configure/FreeGas.md [hunk 2]
```diff
@@ -12,15 +12,21 @@ The miner (or validator in PoA networks) that includes the transaction in a bloc
...
+    [EIP-1820](https://eips.ethereum.org/EIPS/eip-1820) must transfer a small amount of gas (0.08
+    ether) to the single-use deployment account described in the EIP.
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/FreeGas.md:28]: ether -> Ether

File: docs/HowTo/Configure/FreeGas.md (hunk 1 above)
Comment (grantnoble) [reply docs/HowTo/Configure/FreeGas.md:26]: Done.

File: docs/HowTo/Configure/FreeGas.md (hunk 2 above)
Comment (grantnoble) [reply docs/HowTo/Configure/FreeGas.md:28]: Done.

File: docs/HowTo/Configure/FreeGas.md [hunk 3]
```diff
@@ -12,15 +12,21 @@ The miner (or validator in PoA networks) that includes the transaction in a bloc
...
 !!! tip
 
     We use the term _free gas network_ to refer to a network with a gas price of zero. A network
     with a gas price of zero is also known as a _zero gas network_ or _no gas network_.
 
+!!! note
+
+    Networks implementing the smart contract in [EIP-1820](https://eips.ethereum.org/EIPS/eip-1820)
+    must transfer a small amount of gas (0.08 Ether) to the single-use deployment account described
+    in the EIP.
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/FreeGas.md:28]: Words from Rob to make this more general: 
Some pre-crafted transactions require the deployment account to have gas available. For example, the transaction that creates the smart contract in [EIP-1820](https://eips.ethereum.org/EIPS/eip-1820).

Comment (grantnoble) [reply docs/HowTo/Configure/FreeGas.md:28]: Done.

**Pull request merged**